docs/WIRELESS.md

```diff
@@ -158,6 +158,258 @@ The fact that EVE itself stays out of Firmware upgrade business means that you m
+EVE will restart the periodic radio status updates and will POST the outcome of the operation
+back to the application via the radio endpoint.
+
+A formal definition for the syntax and the semantics of the radio endpoint can be found
+[here](../api/PROFILE.md#Radio)
+
+#### Persistence
+
+The intended radio configuration received from Local profile server is persisted
+by EVE (under `/persist` partition) and remains in effect even if the application
+restarts or the edge node reboots (but please see [risks and limitations](#risks-and-limitations)
+associated with the edge node reboot).
+Only once the Local profile server is un-configured or un-deployed will EVE discard
+the last radio config and restore the default behavior of disabled radio silence.
+
+#### Controller connectivity
+
+While the radio silence is imposed, it is perfectly normal for the edge node to lose access
+to the controller, especially if there are no wired connectivity alternatives.
+Normally, this could cause the network configuration to fallback to a previous state
+in an attempt to restore the connectivity. However, in this case this behavior is not desirable,
+therefore the network connectivity testing and the whole fallback mechanism is disabled
+during the radio silence.
+
+Please note that there is also a timer inside EVE to reboot the device if it has not had
+controller connectivity for a prolonged time period - by default this is configured to one week.
+Since it is not expected that a temporary radio silence would remain turned ON for such an extended
+period of time in practice, this timer remains enabled even when radio communication is disabled.
+
+#### Expected application behaviour
+
+Application acting as a Local Profile Server for the radio management is expected
+to behave as follows:
+
+* Application provides UI on the user-side and runs HTTP server with `/api/v1/radio`
+  endpoint on the EVE-side.
+* The application UI should allow to:
+  * Toggle the radio silence intended state (ON/OFF) and as a result change the message content
+    that the HTTP server will respond with to `/api/v1/radio` POST calls.
+  * Show the last received actual state and the last error if there was any
+    (periodically POSTed to `/api/v1/radio` by EVE).
+  * Show that an operation of changing the radio state is still ongoing (e.g. a "loading gif").
+    This starts from a moment of user changing the intended state in the UI, continues
+    through the next POST API call (from which EVE learns the new intended state,
+    different from the actual state), up to the second next POST API call with the actual
+    radio state after making the attempt to apply the new intended config
+    (please remember that for simplicity's sake there will be no POST API calls while
+    a change is ongoing).
+
+#### Radio silence indication
+
+When radio silence is temporarily imposed, the LED (by default the disk LED) will repeat
```

Comment:
Maybe we should call this "the device status indicator (by default the disk LED)" since we know use a second LED on one model. In the future we might also get support for LCDs when it wouldn't be a blinking pattern any more but we can cross that bridge should we get to it.

Reply:
done